05-java-mqtt/src/main/java/demo/ProtelisConfigSpec.java

```diff
@@ -0,0 +1,60 @@
+package demo;
+
+import com.uchuhimo.konf.ConfigSpec;
+import com.uchuhimo.konf.RequiredItem;
+import demo.data.MqttProtelisNode;
+
+import java.util.List;
+
+/**
+ * Konf specification for the TOML configuration file.
+ */
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+public final class ProtelisConfigSpec {
+
+    /**
+     * Create a new config specification named protelis.
+     */
+    public static final ConfigSpec SPEC = new ConfigSpec("protelis");
+    /**
+     * Required field iterations. It represents the number of cycles the simulation will perform.
+     */
+
+    @SuppressWarnings("PMD.FieldNamingConventions")
+    // CHECKSTYLE: ConstantName OFF
+    public static final RequiredItem<Integer> iterations =
+            new RequiredItem<Integer>(SPEC, "iterations") { };
+    /**
+     * Required field protelisModuleName. It is the name of the .pt source file.
+     */
+    @SuppressWarnings("PMD.FieldNamingConventions")
+    // CHECKSTYLE: ConstantName OFF
```

Comment:
I think you just need to suppress checkstyle ONCE, and it remains disabled until you turn it back on. Why is this line copied so many times?